cli/build.gradle.kts

```diff
@@ -131,6 +132,8 @@ dependencies {
     implementation("com.github.ajalt.clikt:clikt:$cliktVersion")
     implementation("org.apache.logging.log4j:log4j-core:$log4jCoreVersion")
     implementation("org.apache.logging.log4j:log4j-slf4j-impl:$log4jCoreVersion")
+    implementation("org.eclipse.sw360.antenna:model:$antennaVersion")
```

Comment:
Just double-checking: Is this really needed? What for? I'm asking because ideally the sw360client library should be usable without Antenna.

Reply:
I need the antenna model dependency for the `ServiceFactory` class to create the `SW360Connection` (see: https://github.com/oss-review-toolkit/ort/blob/f51c20b88684cbe3b98182599dd05f721bdd35e9/cli/src/main/kotlin/commands/UploadResultToSw360Command.kt#L126-L140)

Reply:
Hmm, to me this means the sw360client library is tied to Antenna, which IMO it shouldn't be. Is there a way around this @oheger-bosch, so we only depend on the sw360client library?

Reply:
ServiceFactory is a convenience class to manage shared services during an Antenna run. You can create the HTTP client and the Json object mapper required by the configuration manually. For the HTTP client, there is a factory class _HttpClientFactoryImpl_; the Json mapper could probably be reused from ORT if the configuration is compatible.

Reply:
Then this sounds like the way forward to me.